.github/workflows/ci.yml

```
@@ -39,7 +39,7 @@ jobs:
 
       - name: Install dependencies
         if: steps.cache.outputs.cache-hit != 'true'
-        run: yarn
+        run: yarn && yarn dev:prepare
```

This should be pnpm now no? (with `corepack enable` step before)
```suggestion
        run: pnpm enable && pnpm install && pnpm dev:prepare
```

---

Thanks! Updated this script.